### ISSUE TITLE
Accessibility improvements suggested by ITAO

### ISSUE THREAD

**Issue by adorayi** [cds-snc/design-gc-conception#1297]
One thing that should be considered in the testing is to check for compatibility with the Operating System’s High Contrast mode, for example Windows High Contrast Mode. All text should have at least 4.5:1 colour contrast ratios and all non decorative images (like icons that communicate information) should have at least 3:1 colour contrast ratios in high contrast mode.

As well, it is important to allow user to make certain modifications, like adjusting text spacing.

It would also be good to consider more flexibility in the design of certain components, perhaps as “advanced” settings to allow for edge cases.

For example, we know that headings of the same level should look the same and that their size should also match with their heading levels. The GCDS does this and this is what most people should use.

However, there may be some cases where the visual design needs to be modified, for example an <h2> should look like an <h5>. WET4 classes permitted headings to look different from the default heading level styles, but GCDS does not.
